### ISSUE TITLE
A11y Initiative: Breaking the a11y knowledge gap

### ISSUE THREAD

**Issue by axelavargas** [grafana/grafana#44895]
Currently, the majority of users that create or contribute to Grafana lack knowledge regarding accessibility. The following list describes common problems they face every day:

- Where do I start acquiring knowledge of a11y if I am a <Developer, Contributor, UX>? (Documentation? Course? Checklist?) 
- What are the minimum requirements to have in mind when developing a feature to make it accessible?
- Who can help with reviews of PR’s from a11y point of view?
- What libraries do we have available to build accessible components and not reinvent the wheel?
- Where to ask a11y questions? 

Design Doc: https://docs.google.com/document/d/1jJ_KLq6fhrEdQLQxFnDwtMfUOMhJIFi1pfiJZiem2Zw/edit?usp=sharing
- [x] #46880 
- [ ] #46883
- [ ] Hunting Sessions 
- [ ] Organize team workshops 
- [ ] Create PR Template with a11y focus
- [ ] #44896
